src/main/java/com/kruize/environment/docker/DockerEnvImpl.java

```diff
@@ -256,7 +256,7 @@ private MetricsImpl getMetrics(JsonElement container) throws NullPointerExceptio
         }
 
         if (container.getAsJsonObject().has("cpu_limit")) {
-            containerMetrics.setOriginalMemoryLimit(container.getAsJsonObject().get("cpu_limit").getAsDouble());
+            containerMetrics.setOriginalCpuLimit(container.getAsJsonObject().get("cpu_limit").getAsDouble());
```

Comment:
@dinogun @chandrams Found out what was causing original memory limit not showing up for docker on the dashboard.

The wrong setter is being called, `setOriginalMemoryLimit` instead of `setOriginalCpuLimit`, and it has been that way since Docker support was added in the PR https://github.com/kruize/kruize/commit/64b223fb296915cdb2dce01ed8203286c704c28b#diff-f2f941ba783eebc51fbb5e439a5a0bba

This was causing the changes to be overwritten immediately, and the dashboard would stop showing the memory limits.
I ran petclinic using `docker run --memory="512M" with this change, and it showed up on the dashboard